Resources/Computing/GlobusComputingElement.py

```diff
@@ -0,0 +1,283 @@
+########################################################################
+# File :   GlobusComputingElement.py
+# Author : A.S.
+########################################################################
+
+""" Globus Computing Element
+
+   Allows direct submission to Globus Computing Elements with a SiteDirector Agent
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Resources.Computing.ComputingElement          import ComputingElement
+from DIRAC.Core.Utilities.Grid                           import executeGridCommand
+from DIRAC                                               import S_OK, S_ERROR
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry   import getGroupOption
+from DIRAC.FrameworkSystem.Client.ProxyManagerClient     import gProxyManager
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB     import PilotAgentsDB
+
+
+import os, tempfile
+
+CE_NAME = 'Globus'
+MANDATORY_PARAMETERS = [ 'Queue' ]
+
+class GlobusComputingElement( ComputingElement ):
+  """Globus computing element class
+  implementing the functions jobSubmit, getJobOutput """
+
+  #############################################################################
+  def __init__( self, ceUniqueID ):
+    """ Standard constructor.
+    """
+    ComputingElement.__init__( self, ceUniqueID )
```

Comment:
In this case pylint complains about "using super in an old-style class", because "ComputingElement" does not inherit from "object".
Is it safe to use super here then?